# goal definition
string status_message
string episode
---
bool success